CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## 2.17.0-rc.0 / 2020-03-11
+
+This release implements isolation in TSDB. API queries and recording rules are
+guaranteed to only see full scrapes and full recording rules. This comes with a
+certain overhead in resource usage. Depending on the situation, there might be a
+noticeable increase in memory usage, CPU usage, or query latency.
```

Comment:
```suggestion
some increase in memory usage, CPU usage, or query latency.
```

Reply:
@beorn7 WDYT?

Reply:
I'm a fan of "underpromise, overdeliver", and I found my wording already quite cautious (with "might" and such), but I don't feel strongly.

But if you do want to change "noticeable" to "some", then you have to delete the "a" in the previous line.